pldmtool/pldm_platform_cmd.cpp

```diff
@@ -1397,10 +1397,8 @@ class GetNumericEffecterValue : public CommandInterface
 
         if (rc != PLDM_SUCCESS || completionCode != PLDM_SUCCESS)
         {
-            std::cerr << "Response Message Error: "
-                      << "rc=" << rc
-                      << ",cc=" << static_cast<int>(completionCode)
-                      << std::endl;
+            lg2::error("Response Message Error: rc={RC} cc={CC}", "RC", rc,
```

Comment:
can we have the namespace added and remove lg2::error() and keep error() for uniformity?

Reply:
removed lg2:: tag